### PR TITLE
corretion_legal-space

### PR DESCRIPTION
**Pull request by FrancoisSalmon**

### CODE REVIEW

**Review by ValentinCop — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
ok

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: package.json [hunk 1]
```diff
@@ -17,5 +17,9 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "axios": "^0.26.1",
+    "react-router-dom": "^6.3.0"
```
Comment (LoicBrassart) [inline package.json]: Nope, these packages should only be in the frontend

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
*Meepp* :p

**Review by ValentinCop — APPROVED** [review]

**Review by ValentinCop — DISMISSED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok

**Review by LoicBrassart — APPROVED** [review]
Much better!

**Review by ValentinCop — APPROVED** [review]

**Pull request merged**